### ISSUE TITLE
help with config

### ISSUE THREAD

**Issue by AceCuba** [fristhon/telminal#9]
hey greate bot , cani ask qhere can i store the config.json , becouse every tie i run telminal i need to log the toqen on telegram and thats a little problematic in a place with powerout situations lol . and any whay to put telminal to sealf start when server powers on thanks

**Comment by AceCuba** [comment]
ok i figure out the config.json ,, any way to self boot the telminal when the server is power on

**Comment by fristhon** [comment]
@AceCuba Yes you can use a tool like systemd to run the bot on system startup.
